migrations/1552917482_atomic_swap_ether.js

```diff
@@ -0,0 +1,5 @@
+const AtomicSwapEther = artifacts.require("./AtomicSwapEther.sol");
+
+module.exports = function(deployer) {
+  deployer.deploy(AtomicSwapEther);
```

Comment:
This looks like all the examples. However, deployer.deploy must be asynchonous. This the caller of 1552917482_atomic_swap_ether.js cannot know when the deployment is finished. Strange